pubsub/mock_pub_sub_client_test.go

```diff
@@ -0,0 +1,480 @@
+// Code generated by mockery. DO NOT EDIT.
```

Comment:
This mock and the subscribe mock in this folder are the same as the ones in pubsub/source, so likely you can remove the pubsub/source folder